composer.json

```diff
@@ -11,7 +11,8 @@
     ],
     "require": {
         "roave/security-advisories": "dev-master",
-        "zendframework/zend-expressive": "^0.5"
+        "zendframework/zend-expressive": "^0.5",
+        "zendframework/zend-stdlib": "^2.7"
```

Comment:
2.4 is the minimum supported version. By default composer will retireve the latest (2.7) so I don't see why bum the minimum so high.